do not nested-queue QNodes (#3509)
PennyLaneAI/pennylane: doc/releases/changelog-dev.md

<h3>Breaking changes</h3>

> * Operations executed within a QNode are no longer queued to surrounding contexts.

Reviewer: ```suggestion
* The tape constructed by a QNode is no longer queued to surrounding contexts.
```

Think that might be a little clearer.
